imap-proto/src/parser/rfc3501/mod.rs

```diff
@@ -10,7 +10,7 @@
 use crate::{
     parser::{
         ParseResult,
-        core::{astring_char, astring_utf8, atom_char, atom, number, quoted_utf8, nil, nstring, nstring_utf8, text},
+        core::*,
```

Comment:
And here.

Reply:
Generally, I also prefer explicit imports. But core feels like a prelude as most modules will probably need nearly * from core anyway. I will revert it for now.